### PR TITLE
Crisis adjustments

### PR DESCRIPTION
**Pull request by krpaulson**
This PR represents the progress that I've made to crisis adjustments so far.

TO-DO:

1. Finalize implementation of crisis adjustments for Ebola countries, using population downloaded and prepped by point-people for those countries.
2. Get pre-crisis-adjustment national probabilities of death from UN team (we have death counts not probabilities for this), to be used for our benchmarking.
3. Revisit Myanmar cyclone adjustment from last round.
4. Code for applying Haiti excess deaths (should be fairly quick but requires population input to be run).

### CODE REVIEW

**Comment by jlgodwin** [comment]
Katie and I met and will divide up tasks as follows:

1. Jessica will write a script to get 0-1 and 1-5 population for each of the crisis adjustment countries by both admin1 and admin2
2. Jessica will work on creating crisis_MMR.rda with excess deaths from Cyclone Nargis
3. Katie will work on creating crisis_HTI.rda for the earthquake in Haiti, and update it to apportion deaths from Admin 1 to Admin 2 by population once Jessica completes Step 1.
4. Katie will disaggregate excess deaths from Liberia Admin 1's to Admin 2's based on the population files Jessica produces pop files from Step 1
5. Katie will add code to the BB8.R file or a separate post-benchmarking script to add the excess deaths to the benchmarked estimates

**Comment by alanamcgovern** [comment]
This look great! For the last item, please add code to a new script that will be used after BB8.R. Thank you!

**Comment by krpaulson** [comment]
@alanamcgovern @jlgodwin I think I've addressed all of the items listed for me to do, with the edits I just pushed here to `crisis_deaths_to_qx.R`. I left a few TODOs where I am looking for feedback or input from one of you -- can you review that file on this pull request? Let me know if you have any questions.

**Comment by alanamcgovern** [comment]
@krpaulson Thanks, Katie! No question for now. As for your TODOs, I will add all the filenaming (to load in and to save) once you guys are done with the rest. I think those were the only TODOs for me to provide input on -- but let me know if you need my input on something else.

**Comment by jlgodwin** [comment]
@krpaulson All of the admin weights file you should need have been sent in that email thread. Note for item 4. Liberia's Admin2 -within- Admin1 weights can be calculated from what you receive:

- Admin1 Weight = Admin 1 Pop/Total Pop
- Admin2 (within that Admin 1) Weight = Admin 2 Pop/Total Pop
- So, Admin 2 Weight / Admin 1 Weight = Admin 2 Pop/Admin 1 Pop

I will let you all know when I finish Myanmar's adjustment.

**Comment by alanamcgovern** [comment]
What's going on this? Should I merge the pull request?

**Comment by krpaulson** [comment]
Not yet, I received some population files from folks and I need to test the code, plus need variable names from you for the post-benchmarking files.

File: CrisisAdjustment/crisis_deaths_to_qx.R [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+  deaths_adm2 <- data.frame(
+    country = "Liberia", level = "admin2",
+    gadm = admin2.names$GADM,
+    region = admin2.names$Internal
+  )
+  gadm <- rgdal::readOGR(
+    dsn = "CrisisAdjustment/gadm41_LBR_shp",
+    layer = "gadm41_LBR_2"
+  )@data[, c("NAME_1", "NAME_2")]
+  # TODO: does Liberia have multiple admin2 with same name? If yes the following 2 merges will be bad.
```
Comment (krpaulson) [inline CrisisAdjustment/crisis_deaths_to_qx.R:121]: @alanamcgovern Do you remember how we fixed this issue in the past? The merge needs to happen on the region ID (e.g. `admin2_4` style) but I don't remember how those variables are generated onto the `gadm` object here.

Comment (alanamcgovern) [reply CrisisAdjustment/crisis_deaths_to_qx.R:121]: I can fix that.

**Pull request merged**